orchestra/tests/test_todos.py

```diff
@@ -24,7 +24,8 @@
 
 def _todo_data(task, description, completed,
                skipped_datetime=None, start_by=None,
-               due=None, parent_todo=None, template=None, qa=None):
+               due=None, parent_todo=None, template=None,
+               activity_log=str({'actions': []}), qa=None):
```

Comment:
do you mean `json.dumps`?

Reply:
json.dumps it is.

Reply:
If you can find an example of this being the right way to do things™, I'd be down. If you can't I'd say it's not worth worrying about too much given it's in tests :)

Reply:
Cant find an example. As you say, I wouldn't worry about it. :)